source/apps/crawl_space/views.py

```diff
@@ -335,7 +335,7 @@ class DeleteCrawlView(SuccessMessageMixin, ProjectObjectMixin, DeleteView):
 
     def delete(self, request, *args, **kwargs):
         """ Remove crawl folder """
-        shutil.rmtree(self.get_object().get_crawl_path())
+        shutil.rmtree(self.get_object().crawl_location)
```

Comment:
Can we get a little more protection around this function call?  `shutil.rmtree` is equivalent to `rm -rf` and I'd like some validation that it's running this function in the appropriate subdirectory.